monitoring/mock_uss/interaction_logging/interactions.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+from typing import List
+
+from implicitdict import ImplicitDict
+import yaml
+from yaml.representer import Representer
+
+from monitoring.monitorlib.fetch import Query
+
+
+class QueryDirection(str, Enum):
+    Incoming = "Incoming"
+    """The query originated from a remote client and was handled by the system reporting the interaction."""
+
+    Outgoing = "Outgoing"
+    """The system reporting the interaction initiated the query to a remote server."""
+
+
+class Interaction(ImplicitDict):
```

Comment:
Would it not be better to keep this class in monitorlib, as it would be used by uss_qualifier when it GETs the interaction logs?

Reply:
Interactions are a concept specific to this mock_uss feature -- uss_qualifier can import this file directly when it wants to interact with this mock_uss feature.

Reply:
Ok, I am good if we can import from mock_uss into uss_qualifier.